nbs/core.ipynb

```diff
@@ -35,7 +35,7 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #305.                    f"<code>df</code> must be a list of parquet files, or a pandas, polars or spark DataFrame or <code>None</code>, got: {type(df)}"
</pre><p>Please move the list of files to the end to match the docstring (pandas, polars or spark DataFrame, or a list of parquet files containing the series)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>